llvm/test/SYCLLowerIR/esimd_acc_ptr.ll

```diff
@@ -0,0 +1,37 @@
+; RUN: opt -LowerESIMD -S < %s | FileCheck %s
+
+; This test checks that LowerESIMD pass correctly interpretes the
+; 'kernel_arg_accessor_ptr' metadata. Particularly, that it generates additional
+; vector of per-argument metadata (accessible from "genx.kernels" top-level
+; metadata node):
+; - for those arguments having '1' in the corresponding
```

Comment:
The code seems to allow any non-zero value. Please, either update the comment or add an assert (or some other check) that FE emitted expected value.

Reply:
implemented the nit